.github/workflows/deploy-cloud-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy Cloud Workflow
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "data-pipeline.workflows.yaml"
+  workflow_run:
+    workflows: ["Deploy Dataflow Flex Template"]
+    types: ["completed"]
+
+jobs:
+  deploy:
+    name: Deploy Cloud Workflow
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
```

Comment:
Why is the project ID secret here, but used openly in other scripts?

Reply:
Standard practice. Typically project IDs should not be checked into source control, but since the entire project is public it's not that important here. Either style is fine and we can remove this secret if that's easier to manage/grok.

Reply:
Makes sense. Not harder to manage so we can keep it as is.